src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -44,7 +44,7 @@
             + "but you will not be able to edit existing responses or submit new responses";
 
     // feedback action strings
-    private static final String FEEDBACK_ACTION_SUBMIT = "submit";
+    private static final String FEEDBACK_ACTION_SUBMIT = "submit (or to edit/view)";
```

Comment:
- Note that by changing the string, the variable name `FEEDBACK_ACTION_SUBMIT` is no longer accurate.
- Phrase it as `submit, edit, or view` instead

Reply:
Done!

Reply:
@Vaggelis-Malandrakis the second point is not addressed yet.

Reply:
@wkurniawan07 What do you mean is not addressed? Sorry if I make silly questions. I am new to programming. I changed the variable's name everywhere in the class (from FEEDBACK_ACTION_SUMBIT to FEEDBACK_ACTION_SUBMIT_EDIT_OR_VIEW) as you suggested

Reply:
> Phrase it as `submit, edit, or view` instead

The second point. I was referring to the words, not the variable name, there.

Reply:
@wkurniawan07 I think now it's done!

Reply:
I have to run again the tests using God Mode so that's why the Travis check doesn't work